terraform-modules/aws/msk/test/terratest_test.go

```diff
@@ -0,0 +1,117 @@
+package test
```

Comment:
Remove this file

Reply:
Actually remove the entire `test` directory

Reply:
removed